system-addon/content-src/components/TopSites/_TopSites.scss

```diff
@@ -326,13 +321,54 @@
 }
 
 .topsite-form {
+  .form-input-container {
+    display: flex;
+    max-width: 492px;
+    margin: 0 auto;
+    padding: 32px;
+
+    .top-site-outer {
+      width: 96px;
+      padding: 0;
+      margin: 64px 0 0;
+      margin-inline-start: 32px;
+      list-style: none;
+
+      a {
+        cursor: default;
+      }
+    }
+  }
+
+  label {
+    font-size: 13px;
+  }
+
   .form-wrapper {
-    margin: auto;
-    max-width: 350px;
-    padding: 15px 0;
+    width: 300px;
 
     .field {
       position: relative;
+
+      .clear-input-value {
+        position: absolute;
+        top: 16px;
+        width: 24px;
```

Comment:
Is the click target intended to be wider than the image?

Reply:
No. I changed the markup to have a white background container for the image. The container has `pointer-events: none` so that only clicking the image will clear the input value.